school-library/teachers/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require '../person/persons'
+
+class Teachers < Person
+  def initialize(specialization)
+    super(name, 18, true)
```

Comment:
- [ ] The parameters here should be the parameters given in the `Person` class.e.g. `super(age, name, parent_permission)`. Kindly fix this.